content/en/role-based-access-control.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Role-Based Access Control (RBAC)
+status: Feedback Appreciated
+category: concept
+---
+
+## What it is
+
+The method of regulating systems and resources access based on individual users' roles within an organization.
+
```

Comment:
```suggestion
Each job function within an organization has a different role with specific permissions. 
For instance, a marketing team member may have permission to view the deals sales has in the pipeline (*view* permission) but won't be allowed to edit them (*write* permission). 
Sales and marketing have different permissions based on their roles, hence role-based access control. 

```

Reply:
I think an example would help here.
View/write permission might to be the right term. Can you help me out @iamNoah1?

Reply:
@CathPag I like your edit and I think it's more read/write than view/write.